ts/src/base/Exchange.ts

```diff
@@ -1601,6 +1611,36 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchTradesWs() is not supported yet');
     }
 
+    async watchLiquidations (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}) {
+        throw new NotSupported (this.id + ' watchLiquidations() is not supported yet');
+    }
+
+    async watchAllLiquidations (symbols: string[], since: Int = undefined, limit: Int = undefined, params = {}) {
+        throw new NotSupported (this.id + ' watchAllLiquidations() is not supported yet');
+    }
+
+    async watchLiquidationsForSymbols (symbols: string[], since: Int = undefined, limit: Int = undefined, params = {}) {
```

Comment:
@pcriadoperez can we add proper return types using the liquidation interface? (we should try to add this type in python as well)